forge/containers/wrapper.js

```diff
@@ -13,13 +13,29 @@ module.exports = {
     },
     create: async (project, options) => {
         let value = {}
+        if (this._app.license.get('ee') && this._app.billing) {
```

Comment:
same line I spot a few lines down. Might be an idea to create `this._app.billing_enabled()`?

Reply:
Yes, some way to centrally identify enabled feature flags is probably a good idea

Reply:
This is where we need to step back from billing and think in terms of CE vs EE. There are some aspects of this we simply don't want to iterate on if all it takes is an hour of design work to get something in place that works beyond just one feature.

For large items like this, it would be helpful to have more of an overview of the key changes (API end points, license structure changes, DB models etc) in the PR description.